library/src/main/java/io/getstream/chat/android/client/notifications/handler/ChatNotificationHandler.kt

```diff
@@ -19,13 +19,18 @@ import androidx.core.app.RemoteInput
 import com.google.firebase.messaging.RemoteMessage
 import io.getstream.chat.android.client.R
 import io.getstream.chat.android.client.events.ChatEvent
+import io.getstream.chat.android.client.logger.ChatLogger
 import io.getstream.chat.android.client.notifications.DeviceRegisteredListener
 import io.getstream.chat.android.client.notifications.FirebaseMessageParser
 import io.getstream.chat.android.client.notifications.FirebaseMessageParserImpl
 import io.getstream.chat.android.client.notifications.NotificationLoadDataListener
 import io.getstream.chat.android.client.receivers.NotificationMessageReceiver
 
-open class ChatNotificationConfig(val context: Context) {
+open class ChatNotificationHandler(
```

Comment:
I think it would be good to add @JvmOverloads annotation because otherwise, a constructor with default NotificationConfig won't be existing in Java